nservicebus/operations/metrics/service-control_ttbr_metricsservicecontrol_[1.2,3).partial.md

```diff
@@ -0,0 +1,5 @@
+### Time To Be Received
+
+By default, messages sent to the monitorint instance of Service Control have Time To Be Received set to 7 days. This value can be overriden when needed using following API:
```

Comment:
typo: monitorint

Reply:
We use the most efficient encoding, so I tried to introduce `monitor int`, the new primitive type. Jokes aside, will fix.